CONTRIBUTING.md

```diff
@@ -1,94 +1,125 @@
- for images and other static content.
+The content inside the **docs** folder is organized into sections that are reflected in our Table of Contents (TOC). 
+See where your article fits in the TOC and navigate to the folder associated to that section.
+That folder contains the Markdown files for all articles in the section.
+If necessary, create a new folder to place the files for your content. The main article for that section will be the index.md. 
+For images and other static resources, create a subfolder called **media** inside the folder that contains your article, if it doesn't already exist. Inside the **media** folder, create a subfolder with the article name (except for the index file).
 
-### Example structure ###
+Larger samples should be included in the samples folder under the root of the repo. 
+
+### Example structure
 
     docs
-      /concepts
-      /getting-started
-      /porting
-        /_static
-          portability_report.png
-	  ...
+      /about
+      /core
+        /porting
+          porting-overview.md
+          /media
+            /porting-overview        
+                portability_report.png
+      ...
+    samples
+      /core
+        /porting
+          porting_sample.cs
+
+## Process for contributing
+
+**Step 1:** Open an issue describing the article you wish to write and how it
+relates to existing content. Define where the topic will be located in the TOC - this will determine in which folder to place your article. Get approval to write it. 
+
+You can skip this step for small changes.
 
-Author information is kept in the Markdown file itself. Each author should have a link to an online presence for himself/herself. 
+**Step 2:** Fork the `/dotnet/core-docs` repo.
 
-## Process for contributing ##
+**Step 3:** Create a `branch` for your article.
 
-**Step 1:** Open an Issue describing the article you wish to write and how it
-relates to existing content. Get approval to write your article.
+**Step 4:** Write your article. 
 
-**Step 2:** Fork the `/dotnet/core-docs` repo.
+If it's a new topic, you can use [our template file](template.md) as your starting point. It contains our writing guidelines and also explains the metadata required for each article, such as author information.
 
-**Step 3:** Create a `branch` for your article.
+Place the article in the appropriate folder and any needed images in a media folder located in the same folder as the article.
 
-**Step 4:** Write your article, placing the article in its own folder and any
-needed images in a _static folder located in the same folder as the article.
-Be sure to follow the proper Markdown syntax. If you have code samples,
-place them in a folder within the `/samples/` folder.  For writing guidelines, see
-[our style guide](/styleguide.md). Also do not forget to remove the "wrench" icon 
-(🔧) from the TOC and the file heading, if applicable. 
+Be sure to follow the proper Markdown syntax.
```

Comment:
I'll add a link to the template/style guide then since we have a special syntax as well. :smile: